### DISCUSSION TITLE
Using Dirac Kernel to apply values from VectorPostprocessor in 2D

### DISCUSSION THREAD

**Discussion by salaudeen-ya** [idaholab/moose#24320] [Q&A General] [answered]
Hello,

I need to sample along a line/side set in 2D. The sampled values would be applied in another system as a Dirac force. I have been able to do this in 1D by simple transfer, PointValueSampler and using FunctionDiracSource to use the sampled values. How can I do this for a 2D?

**Comment by GiudGiud** [comment, marked as answer]
Hello

There's a linevalueSampler vector postprocessor (VPP) to get the values on the line
Then there is https://mooseframework.inl.gov/source/dirackernels/ReporterPointSource.html
VPP are Reporters in the backend, you just need to pass the  name_of_object slash name_of_vector to access the Reporter Name of a VPP

Guillaume

**Reply by salaudeen-ya** [reply]
Thank you!

Although I figure out that it might lag one iteration especially when the transfer is done using MultiApps
